moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -50,6 +50,8 @@
 #include <boost/thread/mutex.hpp>
 #include <memory>
 
+rclcpp::Logger LOGGER_COLLISION_DETECTION = rclcpp::get_logger("collision_detection.fcl");
```

Comment:
move inside the namespace

Reply:
https://github.com/ros-planning/moveit2/pull/41/commits/a3011f83f7a76ca3856b5e8b490f587b1af43b7d